### PR TITLE
Update ERC-7540: Move to Review

### PR DESCRIPTION
**Pull request by Joeysantoro**
Addresses comments from https://ethereum-magicians.org/t/eip-7540-asynchronous-erc-4626-tokenized-vaults/16153 and transitions the EIP to `Review` status

### CODE REVIEW

**Review by hieronx — APPROVED** [review]

**Comment by Pandapip1** [comment]
@eth-bot rerun

File: ERCS/erc-7540.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Asynchronous ERC-4626 Tokenized Vaults
 description: Extension of ERC-4626 with asynchronous deposit and redemption support
 author: Jeroen Offerijns (@hieronx), Alina Sinelnikova (@ilinzweilin), Vikram Arun (@vikramarun), Joey Santoro (@joeysantoro)
```
Comment (Joeysantoro) [inline ERCS/erc-7540.md:5]:
```suggestion
author: Jeroen Offerijns (@hieronx), Alina Sinelnikova (@ilinzweilin), Vikram Arun (@vikramarun), Joey Santoro (@joeysantoro), Farhaan Ali (@0xfarhaan)
```

**Review by SamWilsn — APPROVED** [review]
Some mixed capitalization in your definitions section bullets.

File: ERCS/erc-7540.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ The existing definitions from [ERC-4626](./eip-4626.md) apply. In addition, this
 
 ### Request Flows
 
-ERC-7540 Vaults MUST implement one or both of asynchronous deposit and redemption Request flows. If either flow is not implemented in a Request pattern, it MUST use the ERC-4626 standard synchronous interaction pattern. 
+[ERC-7540 Vaults](./erc-7540.md) MUST implement one or both of asynchronous deposit and redemption Request flows. If either flow is not implemented in a Request pattern, it MUST use the ERC-4626 standard synchronous interaction pattern.
```
Comment (Pandapip1) [inline ERCS/erc-7540.md:48]:
```suggestion
[ERC-7540 Vaults](./eip-7540.md) MUST implement one or both of asynchronous deposit and redemption Request flows. If either flow is not implemented in a Request pattern, it MUST use the ERC-4626 standard synchronous interaction pattern. 
```

File: ERCS/erc-7540.md [hunk 3]
```diff
@@ -302,6 +310,9 @@ It reduces code and implementation complexity at little to no cost to simply man
 
 Implementing support for [ERC-165](./eip-165.md) is mandated because of the [optionality of flows](#optionality-of-flows). Integrations can use the `supportsInterface` method to check whether a vault is fully asynchronous, partially asynchronous, or fully synchronous, and use a single contract to support all cases.
 
+### Not Allowing Pending Claims to be Fungible
+The async pending claims represent a sort of semi-fungible intermediate share class. Vaults can elect to wrap these claims in any token standard they like, for example ERC-20, [ERC-1155](./erc-1155.md) or ERC-721 depending on the use case. This is intentionally left out of the spec to provide flexibility to implementers.
```
Comment (Pandapip1) [inline ERCS/erc-7540.md:314]:
```suggestion
The async pending claims represent a sort of semi-fungible intermediate share class. Vaults can elect to wrap these claims in any token standard they like, for example ERC-20, [ERC-1155](./eip-1155.md) or ERC-721 depending on the use case. This is intentionally left out of the spec to provide flexibility to implementers.
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Due to a temporary kludge the links cannot be updated

**Review by Pandapip1 — APPROVED** [review]

**Pull request merged**